library/multichoicerecyclerview/build.gradle

```diff
@@ -8,10 +8,10 @@ checkstyle {
 }
 
 ext {
-    PUBLISH_VERSION = '2.4.0'
-    SUPPORT_LIBRARY_VERSION = '24.2.1'
-    BUILD_TOOLS = "25.0.0"
-    TARGET_SDK = 23
+    PUBLISH_VERSION = '2.3.0'
+    SUPPORT_LIBRARY_VERSION = '25.2.0'
+    BUILD_TOOLS = "23.0.3"
```

Comment:
I am not downgrading the build tools i was updating the support library version instead which by the way needs the TARGET_SDK to match 25